packages/app/src/grow/components/PartnerApp.tsx

```diff
@@ -0,0 +1,20 @@
+import clx from "classnames";
+import { type FC, type ReactNode } from "react";
+
+interface Props {
+  children?: ReactNode;
+  className?: string;
+}
+
+const PartnerApp: FC<Props> = ({ children, className }) => (
```

Comment:
I think we have this  info box component elsewhere, too, so maybe give it a more generic name and move it to common?